components/CustomSlider.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+import { Range, getTrackBackground } from "react-range";
+
+const STEP = 1;
+const MIN = 250;
+const MAX = 4800;
+
+class CustomSlider extends Component {
+  state = {
+    values: [250]
+  };
+
+  render() {
+    const { values } = this.state;
+    const formatedValue = values[0].toLocaleString("en").replace(/,/g, ".");
+
+    return (
+      <>
+        <div className="_flex _justify-center _p-4">
+          <Range
+            values={values}
+            step={STEP}
+            min={MIN}
+            max={MAX}
+            onChange={values => this.setState({ values })}
+            renderTrack={({ props, children }) => (
+              <div
+                onMouseDown={props.onMouseDown}
+                onTouchStart={props.onTouchStart}
+                className="_flex _w-full _h-5"
+              >
+                <div
+                  ref={props.ref}
+                  className="_h-1 _w-full _rounded _self-center"
+                  style={{
+                    background: getTrackBackground({
+                      values: values,
+                      colors: ["#112244", "#ccc"],
+                      min: MIN,
+                      max: MAX
+                    })
+                  }}
+                >
+                  {children}
+                </div>
+              </div>
+            )}
+            renderThumb={() => (
+              <div
+                className="_h-6 _w-6  _flex _justify-center items-center _shadow _bg-blue-100 _rounded-full"
+              />
+            )}
+          />
+        </div>
+        <div className="_flex _flex-col">
+          <span className=" _flex _self-end _text-sm _text-gray-300 _font-secondary">
+            Maximum € 4.800
+          </span>
+          <p className="_text-sm _font-primary  _text-black _font-semibold ">
+            {`Aan te vragen krediet limiet: €${formatedValue}`}
```

Comment:
The template string is not needed here.
```suggestion
            Aan te vragen krediet limiet: €{formattedValue}
```